testing/constraints-3.7.txt

```diff
@@ -9,5 +9,4 @@
 httplib2==0.19.0
 google-auth==1.32.0
 google-auth-httplib2==0.2.0
-google-api-core==1.31.5
```

Comment:
For prototyping purposes only, revert once https://github.com/googleapis/python-api-core/pull/638 is merged and released to PyPI